dxcint/dxcint/utils.py

```diff
@@ -76,3 +83,120 @@ def async_retry_wrapper(*args, **kwargs):
         return async_retry_wrapper
 
     return async_retry_inner
+
```

Comment:
Avoid growing `utils` - eventually they become a garbage can type module. Allocate methods to the classes when appropriate. Also could not find usage of some functions in the codebase - check if they are actually used. Delete if not